w06-ruby-on-rails/d2-dawn-ruby-drills/solutions.rb

```diff
@@ -0,0 +1,60 @@
+#Install the httparty gem ```$ gem install httparty```.
+
+#Now require it in a new ruby script file, and use it to call an album search on the word "White" to the spotify API.
+require 'httparty'
+
+response = HTTParty.get('https://api.spotify.com/v1/search?q=White&type=album')
+p response
+
+#Can you require both ```httparty``` and ```awesome_print``` to have the output look nice? (remember just require awesome_print and then use ```ap``` instead of ```p```)
+require 'httparty'
+require 'awesome_print'
+
+response = HTTParty.get('https://api.spotify.com/v1/search?q=White&type=album')
+ap JSON.parse(response.body)
+
+#In the same file, can you write a loop that returns an array of the album names from your search?
+require 'httparty'
+require 'awesome_print'
+
+response = HTTParty.get('https://api.spotify.com/v1/search?q=White&type=album')
+body =  JSON.parse(response.body)
+body["albums"]["items"].each do |a|
+  ap a["name"]
+end
+
+# Guessing Game
+
+def guessing_game
+	puts "Guess a number between 1 and 100"
```

Comment:
Earlier we were using 2 spaces (the standard in ruby) now it switches to tabs.  Please use spaces only.  https://github.com/bbatsov/ruby-style-guide#source-code-layout